frontend/public/icon-white.svg

```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg" version="1.2" baseProfile="tiny" viewBox="5 4.999 90.001 90.001" xml:space="preserve" height="280" width="280" class="icon-icon-0" data-fill-palette-color="accent">
+    <path d="M5.732 50.518l8.75 8.75c0.976 0.977 2.56 0.977 3.535 0L32.5 44.786l14.482 14.482C47.47 59.756 48.11 60 48.75 60s1.28-0.244 1.768-0.732l8.75-8.75c0.977-0.976 0.977-2.559 0-3.535L44.786 32.5l14.482-14.482c0.977-0.976 0.977-2.559 0-3.535l-8.75-8.75c-0.976-0.977-2.56-0.977-3.535 0L32.5 20.214 18.018 5.732c-0.976-0.977-2.56-0.977-3.535 0l-8.75 8.75c-0.977 0.976-0.977 2.559 0 3.535L20.214 32.5 5.732 46.982C4.756 47.958 4.756 49.542 5.732 50.518zM25.518 34.268c0.977-0.976 0.977-2.559 0-3.535L11.036 16.25l5.214-5.214 14.482 14.482c0.976 0.977 2.56 0.977 3.535 0L48.75 11.036l5.214 5.214L39.482 30.732c-0.977 0.976-0.977 2.559 0 3.535L53.964 48.75l-5.214 5.214L34.268 39.482C33.78 38.994 33.14 38.75 32.5 38.75s-1.28 0.244-1.768 0.732L16.25 53.964l-5.214-5.214L25.518 34.268z" fill="#ffffff" data-fill-palette-color="accent"></path>
+    <path d="M85.518 39.482c-0.976-0.977-2.56-0.977-3.535 0L50 71.464 34.268 55.732c-0.976-0.977-2.56-0.977-3.535 0l-8.75 8.75c-0.977 0.976-0.977 2.559 0 3.535l26.25 26.25C48.72 94.756 49.36 95 50 95s1.28-0.244 1.768-0.732l42.5-42.5c0.977-0.976 0.977-2.559 0-3.535L85.518 39.482zM50 88.964L27.286 66.25l5.214-5.214 15.732 15.732c0.976 0.977 2.56 0.977 3.535 0L83.75 44.786 88.964 50 50 88.964z" fill="#ffffff" data-fill-palette-color="accent"></path>
+</svg>
```

Comment:
Nit: file should end with a beeline character

Reply:
what is it ?